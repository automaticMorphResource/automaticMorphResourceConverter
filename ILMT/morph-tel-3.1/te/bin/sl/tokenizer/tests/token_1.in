కానీ ఇక్కడేముంది?
