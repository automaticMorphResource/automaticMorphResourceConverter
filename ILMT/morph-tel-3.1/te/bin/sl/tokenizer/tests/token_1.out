<Sentence id="1">
1	కానీ	unk
2	ఇక్కడేముంది	unk
3	?	unk
</Sentence>

